cfpb/cfgov-refresh
# Removing focus-target.js code because the chrome bug has been fixed

Removing focus target code

## Changes

- Modified code to remove focus target code. This was fixed in https://bugs.chromium.org/p/chromium/issues/detail?id=473657


## Testing

1. Run `./frontend.sh`.
2.  Visit `http://localhost:8000/#main`. 
3. Click tab, and verify that the `Skip to main content` button is visible. 
4. Click the `Skip to main content` button and verify the `#main` div element is scrolled into view.
  

## Screenshots


## Notes

-

## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
